Database.md

```diff
@@ -196,7 +195,7 @@ WHERE
 #### List directory
 
 ```sql
-SELECT uuid, name, archive_uuid, size, is_shared 
```

Comment:
The thing is, the user should have a way to know if his file is being shared. The initial idea was to put an icon alongside the file name in the list view to indicate it. 

According to that thought the `is_shared` would be lacking here [soap/response/File.java](https://github.com/hawks-atlanta/gateway-java/blob/dev/app/src/main/java/gateway/soap/response/File.java )

Of course we can make changes if necessary. It's not written in stone.

Reply:
You're right